core-services/prow/02_config/openshift/lws-operator/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
```

Comment:
shouldn't we configure `backport-risk-assessed` and `cherry-pick-approved` powers here?

Reply:
Good point. Let me confirm.

Reply:
Updated.